e2e-flap.sh

```
@@ -0,0 +1,789 @@
+    done
+    return 1
+}
+
+# Probe continuously and print a timeline, so a service that comes back and then
+# drops out again is visible. One reporter describes exactly that ("back online
+# for a few seconds then they are offline again"), which a single
+# did-it-recover check cannot see. Echoes the number of failed samples that
+# occurred AFTER the first success.
+watch_reachability() {
+    local svc="$1" duration="$2" label="$3"
+    local elapsed=0 timeline="" seen_up=0 drops=0 res
```

Shellcheck (0.11.0)

[warning] res appears unused. Verify use (or export if used externally).

(SC2034)